### PR TITLE
[Chore](runtime-filter) adjust some check and error msg on runtime filter

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes
adjust some check and error msg on runtime filter

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: be/src/runtime/runtime_filter_mgr.cpp [hunk 1]
```diff
@@ -411,7 +411,8 @@ Status RuntimeFilterMergeControllerEntity::merge(const PMergeFilterRequest* requ
         RuntimeFilterWrapperHolder holder;
         RETURN_IF_ERROR(IRuntimeFilter::create_wrapper(&params, pool, holder.getHandle()));
 
-        RETURN_IF_ERROR(cnt_val->filter->merge_from(holder.getHandle()->get()));
+        // prevent error ignored
+        CHECK(cnt_val->filter->merge_from(holder.getHandle()->get())) << "rf merge failed";
```
Comment (yiguolei) [inline be/src/runtime/runtime_filter_mgr.cpp:415]: why check here？ It will core online.

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by mrhhsg — APPROVED** [review]
LGTM

**Pull request merged**